src/renderer/widgets/Transfer/model/form-model.ts

```diff
@@ -0,0 +1,742 @@
+import { createEvent, createStore, combine, sample, restore } from 'effector';
+import { spread } from 'patronum';
+import { createForm } from 'effector-forms';
+import { BN } from '@polkadot/util';
+
+import { walletModel, walletUtils, accountUtils } from '@entities/wallet';
+import { balanceModel, balanceUtils } from '@entities/balance';
+import { networkModel, networkUtils } from '@entities/network';
+import { xcmTransferModel } from './xcm-transfer-model';
+import { NetworkStore } from '../lib/types';
+import type { Chain, Account, Address, PartialBy, ChainId, ProxiedAccount, AccountId } from '@shared/core';
+import {
+  Transaction,
+  transactionBuilder,
+  transactionService,
+  MultisigTxWrapper,
+  ProxyTxWrapper,
+} from '@entities/transaction';
+import {
+  transferableAmount,
+  getAssetId,
+  formatAmount,
+  validateAddress,
+  toShortAddress,
+  toAccountId,
+  toAddress,
+  dictionary,
+} from '@shared/lib/utils';
+
+type FormParams = {
+  account: Account;
+  xcmChain: Chain;
+  signatory: Account;
+  destination: Address;
+  amount: string;
+  description: string;
+};
+
+type FormSubmitEvent = {
+  transaction: {
+    wrappedTx: Transaction;
+    multisigTx?: Transaction;
+  };
+  formData: PartialBy<FormParams, 'signatory'> & {
+    proxiedAccount?: ProxiedAccount;
+    fee: string;
+    xcmFee: string;
+    multisigDeposit: string;
+  };
+};
+
+const formInitiated = createEvent<NetworkStore>();
+const formSubmitted = createEvent<FormSubmitEvent>();
+
+const feeChanged = createEvent<string>();
+const multisigDepositChanged = createEvent<string>();
+const isFeeLoadingChanged = createEvent<boolean>();
+
+const myselfClicked = createEvent();
+const xcmDestinationSelected = createEvent<AccountId>();
+const xcmDestinationCancelled = createEvent();
+
+const $networkStore = restore(formInitiated, null);
+const $isNative = createStore<boolean>(false);
+const $isMultisig = createStore<boolean>(false);
+const $isProxy = createStore<boolean>(false);
+
+const $isMyselfXcmOpened = createStore<boolean>(false).reset(xcmDestinationCancelled);
+
+const $accountBalance = createStore<string[]>(['0', '0']);
+const $signatoryBalance = createStore<string[]>(['0', '0']);
+const $proxyBalance = createStore<string[]>(['0', '0']);
```

Comment:
maybe use more meaningful type instead of `string[]`? 

`[Balance, NativeBalance]` or even `{balance: string, nativeBalance: string}`